### PR TITLE
Login UI

### PR DESCRIPTION
**Pull request by reifocS**

### CODE REVIEW

File: client/utils/rest/auth/index.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}): Promise<UserResponse> {
+  try {
+    const apiResponse = await axios.post(
+      `${apiUrl}/users/login`,
+      { mail, password },
+      { withCredentials: true }
+    );
+    return { user: apiResponse.data, error: null };
+  } catch (e) {
+    console.log(e);
```
Comment (Naedri) [inline client/utils/rest/auth/index.ts:20]: still needed ?

File: client/pages/dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { getUserInfo } from "../utils/rest/auth";
+
+export async function getServerSideProps(context) {
+  const { user } = await getUserInfo(context.req.cookies?.token);
+
+  if (user) {
+    return {
+      props: { user },
```
Comment (Naedri) [inline client/pages/dashboard.tsx:8]: supplementary comma

**Review by Naedri — APPROVED** [review]
great job

File: client/modules/auth/auth-context.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React, { Dispatch, SetStateAction, useMemo, useState } from "react";
+
+type User = {
+  mail: string;
+  id: number;
+};
+
+const UserContext =
```
Comment (reifocS) [inline client/modules/auth/auth-context.tsx:8]: It's not used yet

**Review by ghost — APPROVED** [review]

**Pull request merged**